e2e/cypress/e2e/order.cy.js

```diff
@@ -330,4 +330,12 @@ describe("Order", () => {
         .should("not.have.attr", "url");
     });
   });
+  describe("User has no agencies", () => {
+    it("should show an errormessage when user has no agencies", () => {
+      cy.visit("/iframe.html?id=modal-order--no-user-agencies");
+      cy.contains("Bestil").click();
```

Comment:
det fejler nok ikke lige nu, men jeg kan lide at tilføje `should("be.visible")` mellem contains og click for at gøre testene mindre flaky